lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,33 @@
+<div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%- gettext("Oops!") %></h3>
```

Comment:
I'd prefer not to refer to the "form" -- students shouldn't need to understand our UI elements. What they're trying to achieve is creating a team, so my suggestion "Your team could not created" still stands :-) assuming my guess about when this heading applies is correct. @clrux if we want to give users more guidance about fixing the error, the error body text is probably the better place? We currently try to keep it generic by referring to highlighted fields that need correcting.